modules/redis/src/main/scala/zio/redis/internal/RespCommand.scala

```diff
@@ -18,18 +18,32 @@ package zio.redis.internal
 
 import zio.Chunk
 
-private[redis] final case class RespCommand(args: Chunk[RespCommandArgument]) extends AnyVal {
-  def ++(that: RespCommand): RespCommand = RespCommand(this.args ++ that.args)
+private[redis] final case class RespCommand(name: RespCommandName, args: RespCommandArguments) {
+  def bulkStrings: Chunk[RespValue.BulkString] = name.bulkStrings ++ args.bulkStrings
+}
+
+private[redis] final case class RespCommandName(str: String) extends AnyVal {
+  def bulkStrings: Chunk[RespValue.BulkString] = Chunk.fromArray(str.split(" ").map(RespValue.bulkString))
+}
+
+private[redis] final case class RespCommandArguments(values: Chunk[RespCommandArgument]) extends AnyVal {
+  def ++(that: RespCommandArguments): RespCommandArguments = RespCommandArguments(this.values ++ that.values)
```

Comment:
Is this method used anywhere now?

Reply:
yes it's used to construct `Input` subtype instances